test/powershell/Modules/Microsoft.PowerShell.Diagnostics/Import-Counter.Tests.ps1

```diff
@@ -35,12 +43,12 @@ function SetScriptVars([string]$rootPath, [int]$maxSamples, [bool]$export)
     $script:tsvPath = Join-Path $rootPath "$rootFilename.tsv"
 
     $script:counterSamples = $null
-    if ($maxSamples)
+    if ($maxSamples -and $IsWindows)
```

Comment:
Seems `Get-Counter` don't work on IoT.
`SetScriptVars` is only called from BeforeAll blocks so we can mask it there by `SkipCounterTests`.